content/organization/tech/accounts/index.en.md

```diff
@@ -115,3 +115,117 @@ Check out the section about [account security](/organization/tech/security/)
+* Title of the event.
+* Date of the event.
+* Name of the Posit Cloud space administrator (person responsible for customizing the Posit Cloud space).
+* Email address of the Posit Cloud space administrator.
+* Name of the R-Ladies chapter organizer coordinating the event.
+* Email address of the R-Ladies chapter organizer coordinating the event.
+
+An R-Ladies Posit Cloud administrator will review your request as soon as possible. If you do not receive a response withing 7 days you are welcome to flag to the `posit-cloud` channel on the R-Ladies organizers slack. 
+
+Once the primary event organizer has been granted access, they should be able
+to add other administrative collaborators to the Posit Cloud space.
+
+Please note that the invitation to the Posit Cloud space must be accepted
+within 7 days; otherwise, the invitation needs to be re-sent.
+
+
+2. Edit the About section with information about your workshop. This should include:
+
+* Title of the event.
+* Date of the event.
+* Name(s) of event organizer.
+* Name of the R-Ladies chapter hosting the event.
+* Brief description of the event.
+
+
+![](img/image1.png)
+
+3. Create your project within the Posit Cloud space. Helpful documentation includes:
+
+ * [Posit Cloud Guide](https://posit.cloud/learn/guide)
+ 
+ * [Posit Cloud Essentials | Ep 1: Getting Started](https://www.youtube.com/watch?v=-fzwm4ZhVQQ) (~30 min YouTube recording)
+ 
+ * [Posit Cloud Essentials | Ep 2: Managing Data Projects with Spaces](https://www.youtube.com/watch?v=JdOn8_Co8VI) (~40 min YouTube recording)
+ 
+  * [Posit Cloud Essentials | Ep 3: Speed up your data workflows with Project Templates](https://www.youtube.com/watch?v=ppfYco3Dhkc) (~30 min YouTube recording)
+
+4. Change project access to space members.
+
+![](img/image4.png)
+
+5. Change space access to sharing link with role as Contributor, which allows the 
+event organizer to share a url with participants to join the space. The "Contributor” role
+enables the participants to work in their own personal copy of the project.
+
+![](img/image2.png)
+
+
+6. When users receive a sharing link they will enter the space, they will be required 
+to either login or create a Posit Cloud account. Then users have to click on content 
+to open the project. It may take a couple of minutes for the initial deployment. Then
+users will be invited to make a copy of the project.
+
+![](img/image3.png)
+
+The project copy is labelled as "Private" for the user, indicating that they get
+their own dedicated computing resources, and any changes made in this project
+do not affect other students.
+
+![](img/image10.png)
+
+
+### Posit Cloud Administration
+
+This section is for members of the Global Team who have administrative
+access to the Posit Cloud account. 
+
+1. Create a new space. Name the space `chapter-title` and assign to r-ladies account.
+
+![](img/image5.png)
+
+2. Add email of primary workshop leader with an Admin role on the space.
+
+![](img/image7.png)
+
```

Comment:
What we plan to do with the spaces people create after the workshop is held? Delete them after X days?

Reply:
excellent point, thank you!